### PR TITLE
Add global unique var

### PR DESCRIPTION
**Pull request by ngrundler**
This PR adds a variable so users can enable the configuration setting controlling enforcement of unique IP address space in the global VRF, as well as a handler to restart the netbox app via supervisorctl.

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -14,3 +14,6 @@
     name: supervisor
     state: restarted
   listen: "restart supervisor"
+
+- name: restart netbox
+  supervisorctl: name=netbox state=restarted
```
Comment (ksofa2) [inline handlers/main.yml]: Please adjust to use a YAML dict for this to keep the formatting consistent with the rest of the code.

Comment (ngrundler) [reply handlers/main.yml]: Done.

**Comment by ksofa2** [comment]
@ngrundler Can you resolve conflicts?

**Comment by ngrundler** [comment]
@ksofa2 resolved.

**Pull request merged**